docusaurus/docs/how-to-guides/data-source-plugins/add-support-for-pdc.md

```diff
@@ -0,0 +1,220 @@
+---
+id: add-support-for-pdc
+title: Add support for Private Datasource Connect (PDC)
+description: Add support for private datasource connect (PDC) functionality Grafana plugin development.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - pdc
+  - private datasource connect
+---
+
+# Add support for private datasource connect (PDC)
+
+## What is Private Data Source Connect?
+
+Private data source connect, or PDC, is a way for you to establish a private, secured connection between a Grafana Cloud instance, or stack, and data sources secured within a private network.
+
+Read more about it [here](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/private-data-source-connect/).
+
+### When NOT to support PDC
+
+PDC is a Grafana Cloud - only solution, so if your datasource is not available in Grafana Cloud, there is not much benefit from implementing PDC support.
+
+# Adding PDC to a data source
+
+PDC support must be integrated in each data source because each plugin in Grafana is responsible for establishing its own connection to the target data source. While Grafana stores the proxy configuration details (such as proxy_address, server_address, and certificates) in its config, each plugin consumes this configuration in a different way.
```

Comment:
```suggestion
PDC support must be integrated in each data source because each plugin in Grafana is responsible for establishing its own connection to the target data source. While Grafana stores the proxy configuration details (such as `proxy_address`, `server_address`, and certificates) in its config, each plugin consumes this configuration in a different way.
```